charts/misc-apps/Chart.yaml

```diff
@@ -17,14 +17,8 @@ dependencies:
 annotations:
   artifacthub.io/changes: |
     - kind: changed
-      description: "keda: update from 2.9.3 to 2.10.0"
+      description: |
+        reloader: addition to misc-apps
       links:
-        - name: GitHub Release
-          url: https://github.com/kedacore/keda/releases/tag/v2.10.0
-    - kind: changed
-      description: "keda: update chart from 2.9.3 to 2.10.1"
-      links:
-        - name: "feat: Ship KEDA Core v2.10.1 and align Kubernetes expectations"
-          url: https://github.com/kedacore/charts/pull/407
-        - name: "feat: prepare v2.10"
-          url: https://github.com/kedacore/charts/pull/400
+        - name: GitHub PR
+          url: https://github.com/adfinis/helm-charts/pull/xxx
```

Comment:
no need for self referential linking, please add some links that help discover reloader instead

Reply:
4b2ce88